tubes/routing.py

```diff
@@ -1,4 +1,4 @@
-# -*- test-case-name: tubes.test.test_fan -*-
+# -*- test-case-name: tubes.test.test_routing -*-
 # Copyright (c) Twisted Matrix Laboratories.
 # See LICENSE for details.
 
```

Comment:
This doc string isn't very accurate; we should demonstrate that the message destination must be added to the message before being sent to the router. In `test_twoRoutes` this is made clear with routeDrain constructed like this :

```
routeDrain = series(chooser, router.drain)
```